core/shared/src/main/scala/rapture/core/parser.scala

```diff
@@ -89,6 +89,18 @@ trait StringParser_1 {
 }
 
 object StringParser extends StringParser_1 {
+  def apply[T](f: String => T): StringParser[T] = {
```

Comment:
The return type should be `StringParser[T] { type Throws = ParseException }` -- we lose the `Throws` refinement, otherwise!